docs/components/docs/Navigation.tsx

```diff
@@ -155,6 +155,9 @@ export function DocsNavigation() {
         <NavItem href="/docs/guides/custom-admin-ui-pages">
           Custom Admin UI Pages <Badge look="success">New</Badge>
         </NavItem>
+        <NavItem href="/docs/guides/custom-admin-ui-navigation">
+          Custom Admin-UI Navigation <Badge look="success">New</Badge>
```

Comment:
```suggestion
          Custom Admin UI Navigation <Badge look="success">New</Badge>
```

For consistency with other nav items